01.fizzbuzz/fizzbuzz.rb

```diff
@@ -0,0 +1,13 @@
+numbers = 1..20
+numbers.each do |number|

```

Comment:
今回のようなケースでは`numbers`変数を定義して代入するメリットがあまりないので、Rangeオブジェクトのリテラルから直接`each`を呼び出してしまうと良いと思います。（動いてますし修正はしなくても大丈夫です。次回同じようなことがあったら思い出してください〜）

```suggestion
(1..20).each do |number|
```